ansible/README.rst

```diff
@@ -0,0 +1,30 @@
+Ansible and Docker OVN Emulation Guide
```

Comment:
Give a name to your tool, create a folder  move 'ansible' and 'docker' folder into it. Keeping them together can avoid confuse others.